packages/@react-aria/table/src/useTableColumnHeader.ts

```diff
@@ -27,7 +27,9 @@ export interface AriaTableColumnHeaderProps {
   /** An object representing the [column header](https://www.w3.org/TR/wai-aria-1.1/#columnheader). Contains all the relevant information that makes up the column header. */
   node: GridNode<unknown>,
   /** Whether the [column header](https://www.w3.org/TR/wai-aria-1.1/#columnheader) is contained in a virtual scroller. */
-  isVirtualized?: boolean
+  isVirtualized?: boolean,
+  /** Whether the column has a menu in the header, this changes interactions with the header. */
+  hasMenu?: boolean
```

Comment:
Should this be less specific to menus? Should it be more like an override to disable the default sorting behavior?

Reply:
Maybe, what are the use cases other than a menu where we'd want to disable the default sorting behavior?

Reply:
Not sure, but I mainly wondered if menus were too spectrum-specific?

Reply:
maybe, it was the ask for a future feature, custom menu options on columns. so that's where the name came from. I'm fine changing it though to `disableDefaultSort` or `disableDefaultInteraction`?